app/components/charts/line_chart_component.rb

```diff
@@ -0,0 +1,2 @@
+class Charts::LineChartComponent < ViewComponent::Base
+end
```

Comment:
Thanks! This ended up looking a little different. I was looking to support defaults as well as additional options to be passed in, since Chartkick supports more options and each graph may have different specifics. I'll push it up shortly to see what you think.